src/snovault/elasticsearch/create_mapping.py

```diff
@@ -273,17 +276,14 @@ def audit_mapping():
 
 def es_mapping(mapping):
     return {
-        '_all': {
-            'enabled': True,
-            'analyzer': 'snovault_search_analyzer'
-        },
         'dynamic_templates': [
             {
                 'template_principals_allowed': {
                     'path_match': "principals_allowed.*",
                     'match_mapping_type': "string",
                     'mapping': {
                         'type': 'keyword',
+                        'copy_to': '_all',
```

Comment:
It doesn't make sense to me to copy `principals_allowed.*` to `_all`. I'm actually kind of surprised that the `links` and `unique_keys` fields are stored in `_all` in our current version, if that's what's happening.